src/scf-release/src/acceptance-tests-brain/test-scripts/014_minibroker_mariadb_test.sh

```diff
@@ -0,0 +1,197 @@
+#!/bin/bash
+## # # ## ### Tracing and common configuration ### ## # #
+## Remove CF_ variables not used by the test.
+
+set -o errexit
+set -o xtrace
+
+function random_suffix { head -c2 /dev/urandom | hexdump -e '"%04x"'; }
+CF_ORG="${CF_ORG:-org}-$(random_suffix)"
+CF_SPACE="${CF_SPACE:-space}-$(random_suffix)"
+CF_TCP_DOMAIN="${CF_TCP_DOMAIN:-tcp-$(random_suffix).${CF_DOMAIN}}"
+CF_SEC_GROUP="${CF_SEC_GROUP:-sec-group}-$(random_suffix)"
+CF_BROKER="${CF_BROKER:-minibroker}-$(random_suffix)"
+CF_APP="${CF_APP:-app}-$(random_suffix)"
+CF_SERVICE="${CF_SERVICE:-service}-$(random_suffix)"
+MINIBROKER_REPO="${MINIBROKER_REPO:-https://minibroker-helm-charts.s3.amazonaws.com/minibroker-charts/}"
+KUBERNETES_REPO="${KUBERNETES_REPO:-https://minibroker-helm-charts.s3.amazonaws.com/kubernetes-charts/}"
+HELM_RELEASE_NAME="${HELM_RELEASE_NAME:-minibroker}-$(random_suffix)"
+KUBE_NAMESPACE="${KUBE_NAMESPACE:-minibroker}-$(random_suffix)"
+KUBE_INSTANCE_NAMESPACE="${KUBE_INSTANCE_NAMESPACE:-minibroker-pods}-$(random_suffix)"
+
+## # # ## ### Login & standard entity setup/cleanup ### ## # #
+## Remove operations not relevant to the test
+
+function login_cleanup() {
+    trap "" EXIT ERR
+    set +o errexit
+
+    cf delete-space -f "${CF_SPACE}"
+    cf delete-org -f "${CF_ORG}"
+
+    set -o errexit
+}
+trap login_cleanup EXIT ERR
+
+# target, login, create work org and space
+cf api --skip-ssl-validation "api.${CF_DOMAIN}"
+cf auth "${CF_USERNAME}" "${CF_PASSWORD}"
+
+cf create-org "${CF_ORG}"
+cf target -o "${CF_ORG}"
+
+cf create-space "${CF_SPACE}"
+cf target -s "${CF_SPACE}"
+
+## # # ## ### Test-specific configuration ### ## # #
+## Remove and extend as needed
+
+# Location of the test script. All other assets will be found relative
+# to this.
+SELFDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+
+## # # ## ### Test-specific code ### ## # #
+## For custom cleanup retrap the signals EXIT & ERR to run a custom
+## function, and chain to login_cleanup inside. Remove if not needed.
+
+# We have some waits in this test, and want to clean things up properly
+# (especially the service broker) when we abort.  So in our waits we check that
+# nine minutes haven't elapsed; this gives us around a minute to do any clean up
+# we need.
+DEADLINE="$(date --date=9min +%s)"
```

Comment:
Let's make `9min` a const that is easier to control. If you feel like it should also be shared between other tests, let's move to a shared script. Otherwise, `check_deadline` should take it as a parameter.